.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish to FTP on Development Push
+
+on:
+  push:
+    branches:
+      - development
```

Comment:
we need to run auto deployment for production too. Do you need to create another yaml file for the main branch or do you want to modify this one to work for both development and main?

Reply:
also, add the worflow_dispatch trigger so we can run this manually.
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

Reply:
Done